src/components/admin-message/container.vitest.tsx

```diff
@@ -0,0 +1,51 @@
+import { screen } from '@testing-library/react';
+import { renderWithProviders } from '../../test-utils';
+
+import { AdminMessageContainer } from './container';
+import { AdminMessageType } from '../../store/messages';
+
+describe('AdminMessage', () => {
+  it('should render admin message if isAdmin is true', () => {
+    renderWithProviders(
+      <AdminMessageContainer
+        // @ts-ignore
+        message={{
+          message: 'some message',
+          isAdmin: true,
+          admin: { type: AdminMessageType.JOINED_ZERO, inviterId: 'inviter-id', inviteeId: 'current-user' },
+        }}
+      />,
+      {
+        preloadedState: {
+          authentication: {
+            user: {
+              // @ts-ignore
+              data: { id: 'current-user' },
+            },
+          },
+          normalized: {
+            users: {
+              'inviter-id': { id: 'inviter-id', firstName: 'Courtney' },
+            },
+          },
+        },
+      }
+    );
```

Comment:
I think we may need to iterate on this a bit. I like the way how we currently do it (using `subject({ ..})` as a base function and passing the relevant `props` in the test. 
<img width="972" alt="image" src="https://github.com/zer0-os/zOS/assets/33264364/09c407c5-b2df-4c30-9bdd-4e6cdbeb8778">

That would also be extendable to other tests which we'll migrate, and would be easier to understand (from the perspective on how the current tests are written).

Reply:
Can for sure create a reusable render function. I just didn't do it here as there are only 2 tests - for a larger test suite I'd definitely make a function